server/models/lifecycle.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const LifecycleSchema = new Schema({
+  name: {
+    type: String,
+    trim: true,
+  },
+  issues: [
```

Comment:
please explain :)
why do we need issues in lifecycle

Reply:
Because each lifecycle in a board will have its set of issues.
For eg:
1. _Todo_ lifecycle will have some set of issues
2. _In Progress_ lifecycle will have another set of issues.
One issue can be moved from one lifecycle to another lifecycle.